staff_features/shared/step_definitions.rb

```diff
@@ -22,6 +22,40 @@
   click_on_string string
 end
 
+When 'the user clicks on the dropdown in the {string} form' do |form_title|
```

Comment:
Would a form ever have two dropdowns?

Reply:
It certainly could. Though it would be a bit more verbose, it might be good to rename this as "clicks on the first dropdown" just to make what it is doing clear.

Reply:
Many sub-record forms can have an arbitrary amount of dropdown buttons.

I have already considered this, and I have intentionally left it as it is because, at this point, all the created records have strictly one entry in each sub-record form.
This applies to other fields related to sub-records.

My intention was to elaborate further, once we are going to write features with more than one entry in sub-record forms, to devote more time and focus on a solution on how to effectively implement this.

Probably we are going to use words like `first`, `second`, etc., but numbering has to be scoped to refer to the entry inside a sub-record form because it is going to be cumbersome and error-prone to manually count dropdown buttons and/or fields.

eg.: It is going to be something like:
`the user clicks on the second dropdown in the third entry in the Agent Links form`

My suggestion is to leave it like this for now, to create some headroom for conceiving a clear and concise solution for all sub-records if possible.

Reply:
I'm ok with having just this one step definition that clicks the first entry in the subrecord form, and we can of course consider alternatives in the future as needed. I still would prefer it to be named as such (`clicks on the first dropdown`) just to avoid confusion, especially since it is a shared step definition that may be used in a variety of contexts (including by test authors that may not be familiar with the Capybara end of things).

Reply:
Done